javalanglib/src/main/scala/java/lang/Long.scala

```diff
@@ -44,6 +44,14 @@ object Long {
   final val MAX_VALUE = 9223372036854775807L
   final val SIZE = 64
 
+  /**
+   * Holds the maximal number of digits for each radix, so that a number of
+   * that radix can be converted to an Integer
+   */
```

Comment:
Just the ScalaDoc style:
```scala
  /** Maximal number digits for each radix, so that a number of
   *  that radix can be converted to an Int.
   */
```